var/spack/repos/builtin/packages/geant4-data/package.py

```diff
@@ -38,7 +39,8 @@ class Geant4Data(BundlePackage):
     depends_on("g4emlow@7.13", when='@10.7.0:10.7.9999')
     depends_on("g4photonevaporation@5.7", when='@10.7.0:10.7.9999')
     depends_on("g4radioactivedecay@5.6", when='@10.7.0:10.7.9999')
-    depends_on("g4particlexs@3.1", when='@10.7.0:10.7.9999')
+    depends_on("g4particlexs@3.1.1", when='@10.7.1:10.7.9999')
+    depends_on("g4particlexs@3.1", when='@10.7.0:10.7.0')
```

Comment:
```suggestion
    depends_on("g4particlexs@3.1", when='@10.7.0')
```

I think this can be simplified with the single exact version form.

Reply:
Thanks for the suggestion. Changed!

Reply:
Great! Good to merge once CI passes.

Reply:
Can you confirm, that 10.7.1 needs `g4particlexs@3.1.1`? At least, that's my reading of the web site. (I don't know much about geant4 TBH).

Reply:
Yep, that's the case! The canonical list is always in the `G4DatasetDefinitions.cmake` file, and it's a bug if the website download page conflicts with that.